src/app/public/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -0,0 +1,46 @@
+import { Directive, ElementRef, Input, OnInit, Optional } from '@angular/core';
+import { AutonumericConfig } from './autonumeric-config';
+let autoNumeric: any = require('autonumeric');
+
+@Directive({
+  selector: '[autonumeric]'
```

Comment:
Mind if we prefix the selector with `sky`?

Reply:
Updated this.